### PR TITLE
Updated strings from Crowdin 20220714

### PR DESCRIPTION
**Pull request by mikehardy**
Standard I18N sync - this one is quite large, and is *all* real translation activity (that is: there are no string additions / removals from development work here, it is all translators translating strings)

Needs review

### CODE REVIEW

File: AnkiDroid/src/main/res/values-uz/01-core.xml [hunk 1]
```diff
@@ -161,37 +160,37 @@
...
-    <string name="button_sync" comment="Text of the sync button">Sync</string>
-    <string name="cancel_sync_confirm">Do you want to cancel the sync?</string>
-    <string name="continue_sync">Continue sync</string>
-    <string name="sync_cancelled">Sync cancelled</string>
-    <string name="sync_cancel_message">Cancelling&#8230;\nThis may take some time.</string>
+    <string name="studyoptions_congrats_more_new">Bugungi chegaraga yetdingiz, lekin hali ham takrorlash uchun kartalar bor. Chegarani sozlamalar qismidan oshirishingiz mumkin, lekin bu sizning qisqa vaqtdagi ko\'rib chiqish yukingizni oshiradi.</string>
+    <string name="studyoptions_congrats_custom">Odatiy jadvaldan tashqari ko\'proq o\'qish uchun, \'Qo\'shimcha shug\'ullanish\' tugmasini bosing</string>
+    <string name="studyoptions_no_cards_due">Tugashi lozim kartalar halicha yo\'q</string>
+    <string name="sd_card_not_mounted">Qurilma xotirasi o\'rnatilmagan</string>
+    <string name="empty_cloze_warning">Bu kartani o\'zgartirish va bir nechta yopilma o\'chirishlar qo\'shish</string>
```
Comment (mikehardy) [inline AnkiDroid/src/main/res/values-uz/01-core.xml:178]: this is an error

**Comment by mikehardy** [comment]
let's see what came in the last 4 days...

File: AnkiDroid/src/main/res/values-az/01-core.xml [hunk 2]
```diff
@@ -63,8 +63,8 @@
     <string name="collapse">Daralt</string>
     <string name="study_more">Daha çox öyrən</string>
     <plurals name="deckpicker_title">
-        <item quantity="one">%1$d card due (%2$s)</item>
-        <item quantity="other">%1$d cards due (%2$s)</item>
+        <item quantity="one">2 kart edildi (1dəq)</item>
+        <item quantity="other">3 kart edildi (1dəq)</item>
```
Comment (mikehardy) [inline AnkiDroid/src/main/res/values-az/01-core.xml:67]: nope

File: AnkiDroid/src/main/res/values-az/01-core.xml [hunk 3]
```diff
@@ -75,20 +75,20 @@
         <item quantity="other">%1$d hours %2$d left</item>
     </plurals>
     <plurals name="reviewer_window_title_days_new">
-        <item quantity="one">%1$d day %2$d left</item>
+        <item quantity="one">Gün</item>
```
Comment (mikehardy) [inline AnkiDroid/src/main/res/values-az/01-core.xml:78]: nope

File: AnkiDroid/src/main/res/values-az/01-core.xml [hunk 4]
```diff
@@ -75,20 +75,20 @@
...
     </plurals>
-    <string name="no_cards_placeholder_title">Collection is empty</string>
-    <string name="no_cards_placeholder_description">Start adding cards\nusing the + icon.</string>
+    <string name="no_cards_placeholder_title">Dəstə boşdur</string>
+    <string name="no_cards_placeholder_description">+ işarəsindən istifadə edərək kart əlavə etməyə başlayın</string>
     <!-- The deck picker via AnkiStatsTaskHandler.java -->
     <plurals name="studied_cards_today">
         <item quantity="one">Studied %1$d card in %2$s today</item>
-        <item quantity="other">Studied %1$d cards in %2$s today</item>
+        <item quantity="other">Bugün 0 dəqiqədə 1 kart edildi</item>
```
Comment (mikehardy) [inline AnkiDroid/src/main/res/values-az/01-core.xml:86]: nope

File: AnkiDroid/src/main/res/values-az/01-core.xml [hunk 5]
```diff
@@ -147,31 +147,31 @@
     -->
     <string name="time_quantity_seconds">%d s</string>
     <string name="time_quantity_minutes">%d dəq</string>
-    <string name="time_quantity_hours_minutes">%1$d h %2$d m</string>
+    <string name="time_quantity_hours_minutes">(4 dəq)</string>
```
Comment (mikehardy) [inline AnkiDroid/src/main/res/values-az/01-core.xml:150]: nope

File: AnkiDroid/src/main/res/values-az/01-core.xml [hunk 6]
```diff
@@ -147,31 +147,31 @@
...
     <string name="time_quantity_hours">%d saat</string>
     <string name="time_quantity_days">%d gün</string>
-    <string name="time_quantity_days_hours">%1$d d %2$d h</string>
+    <string name="time_quantity_days_hours">(4 dəq)</string>
```
Comment (mikehardy) [inline AnkiDroid/src/main/res/values-az/01-core.xml:153]: nope

**Comment by david-allison** [comment]
I'm really impressed by the velocity we're seeing. I know I haven't been active, and it's heartwarming to see how quickly things are moving

**Comment by david-allison** [comment]
```
> Task :AnkiDroid:lintPlayDebug FAILED
/home/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/main/java/com/ichi2/libanki/Utils.java:142: Error: Format string 'time_quantity_hours_minutes' is not a valid format string so it should not be passed to String.format [StringFormatInvalid]
            return res.getString(R.string.time_quantity_hours_minutes, (int) Math.floor(time_s/TIME_HOUR), (int) Math.round((time_s % TIME_HOUR) / TIME_MINUTE));
                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /home/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/main/res/values-az/01-core.xml:150: This definition does not require arguments
/home/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/main/java/com/ichi2/libanki/Utils.java:144: Error: Format string 'time_quantity_days_hours' is not a valid format string so it should not be passed to String.format [StringFormatInvalid]
            return res.getString(R.string.time_quantity_days_hours, (int) Math.floor(time_s/TIME_DAY), (int) Math.round((time_s % TIME_DAY) / TIME_HOUR));
                   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /home/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/main/res/values-az/01-core.xml:153: This definition does not require arguments
/home/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/main/res/values/01-core.xml:129: Error: Inconsistent number of arguments in formatting string time_quantity_hours_minutes; found both 2 here and 0 in values-az/01-core.xml [StringFormatCount]
    <string name="time_quantity_hours_minutes">%1$d h %2$d m</string>
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /home/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/main/res/values-az/01-core.xml:150: Conflicting number of arguments (0) here
/home/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/main/res/values/01-core.xml:132: Error: Inconsistent number of arguments in formatting string time_quantity_days_hours; found both 2 here and 0 in values-az/01-core.xml [StringFormatCount]
    <string name="time_quantity_days_hours">%1$d d %2$d h</string>
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    /home/runner/work/Anki-Android/Anki-Android/AnkiDroid/src/main/res/values-az/01-core.xml:153: Conflicting number of arguments (0) here
4 errors, 0 warnings
```

**Comment by mikehardy** [comment]
All fixed upstream as far as I can tell, just waiting on "you can't export faster than NN minutes on free plan" timer to expire upstream on crowdin before re-running action then re-close/re-open here :crossed_fingers: - probably tomorrow morning though, really late here

**Pull request merged**